examples/maps/maps.bal

```diff
@@ -43,6 +43,8 @@ public function main() {
 
     // Note that Index-based access will return a union of constraint+nil.
```

Comment:
Index -> index

Shall we rephrase this to say "Note that the return type of index-based access will be `T?` (where `T` is the constraint type of the map`)"